src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +1,18 @@
 *REMOVED*Microsoft.CodeAnalysis.Operations.IEventAssignmentOperation.EventReference.get -> Microsoft.CodeAnalysis.Operations.IEventReferenceOperation
+Microsoft.CodeAnalysis.INamespaceDeclaration
+Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration
+Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration.Children.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration>
+Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration.IsNamespace.get -> bool
+Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration.IsType.get -> bool
+Microsoft.CodeAnalysis.INamespaceOrTypeDeclaration.Name.get -> string
+Microsoft.CodeAnalysis.ITypeDeclaration
+Microsoft.CodeAnalysis.ITypeDeclaration.Arity.get -> int
```

Comment:
>Arity [](start = 40, length = 5)

I don't recall discussing Arity in the completion scenario. Just want to confirm that is intentional/needed/useful.

Reply:
it's likely necessary because we put things in the completion list like `Foo<> (from Bar.Baz)`.  To place the `<>` we need to know it's generic.  'Arity' is a nice, already existing, property for that.